src/shared-ui/src/assets/icons.html

```diff
@@ -1,29 +1,3 @@
-<!--
```

Comment:
The license was probably removed when I generated these files again with the new icons. It should be added back

Reply:
Fixed